tests/Jenkins/utilities.sh

```diff
@@ -318,13 +313,9 @@ installDIRAC() {
     exit 1
   fi
 
-  if [[ "${CLIENT_USE_PYTHON3:-}" == "Yes" ]]; then
-    if [[ -n "${DIRACOSVER+x}" ]]; then
-      if [[ "${DIRACOSVER:-}" == "latest" ]]; then
-        DIRACOS2_URL="https://github.com/DIRACGrid/DIRACOS2/releases/latest/download/DIRACOS-Linux-x86_64.sh"
-      else
-        DIRACOS2_URL="https://github.com/DIRACGrid/DIRACOS2/releases/download/${DIRACOSVER}/DIRACOS-Linux-x86_64.sh"
-      fi
+  if [[ "${USE_PYTHON3:-}" == "Yes" ]]; then
```

Comment:
what not `${CLIENT_USE_PYTHON3:-}` ?

Reply:
The variables are now automatically stripped of `CLIENT_`/`SERVER_` so you do things like use different DIRACOS versions in the client and server containers or selectively enable other features.

Reply:
Where is this stripping happening?

Reply:
In the Python script when it generates the `CONFIG` files which are sourced:

https://github.com/DIRACGrid/DIRAC/blob/1dc48d11424a031088afcf5380784fbe8fba53dd/integration_tests.py#L194-L205